packages/ozone/src/daemon/scheduled-action-processor.ts

```diff
@@ -192,6 +210,36 @@ export class ScheduledActionProcessor {
     // register the takedown in event pusher
     await moderationTxn.takedownRepo(subject, moderationEvent.event.id)
 
+    if (email.content && email.subject) {
+      let isDelivered = false
+      try {
+        await retryHttp(() =>
+          moderationTxn.sendEmail({
+            ...email,
+            recipientDid: action.did,
+          }),
+        )
+        isDelivered = true
+      } catch (err) {
+        dbLogger.error(
+          { err, did: action.did },
+          'failed to send takedown email',
+        )
+      }
+      await moderationTxn.logEvent({
+        event: {
+          content: email.content,
+          subjectLine: email.subject,
+          $type: 'tools.ozone.moderation.defs#modEventEmail',
+          comment: 'Communication attached to scheduled action',
```

Comment:
Maybe vary the comment based on `isDelivered`? Then you'd be able to clearly see from the comment alone if the email communication was sent correctly or not from the scheduled action.